sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_access_control_client.py

```diff
@@ -109,6 +109,63 @@ def list_role_assignments(self, role_scope, **kwargs):
             **kwargs
         )
 
+    @distributed_trace
+    def set_role_definition(self, role_scope, role_definition_properties, **kwargs):
```

Comment:
Yeah, that's something I'm still taking a look at. Still leaning strongly toward requiring a `KeyVaultRoleDefinitionProperties` instance though, just because of the volume of parameters that would otherwise be required here